src/CrvInfoHelper.cc

```diff
@@ -37,12 +37,34 @@ namespace mu2e
     {
       const CrvCoincidenceCluster &cluster = crvCoincidences->at(i);
 
+      // Get the PEs per layer from the reco pulses
+      float PEsPerLayer_[4] = {0.}; // Four layers, each one initiliased to zero PEs in a static array
```

Comment:
Similar comment to earlier about the bare 4

Reply:
I'll update this with the `nLayers` from the offline geometry.

Reply:
Same as above: You can use `CRVId::nLayers` defined in `Offline/DataProducts/inc/CRVId.hh`.